scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -6,57 +6,37 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+Workload partitioning segregates compute node CPU resources into distinct CPU sets. The primary objective is to keep platform pods on the specified cores to avoid interrupting the CPUs the customer workloads are running on. Workload partitioning isolates {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. This ensures that the remaining CPUs in their deployment remain untouched and available exclusively for their own workloads. The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
```

Comment:
Small nit for the `This ensures that the remaining CPUs in their deployment remain untouched and available exclusively for their own workloads.` sentence. Maybe something like the one below is a little clearer wdyt?

>This ensures that the remaining CPUs in the cluster deployment are untouched and available exclusively for non platform workloads.